docs/conceptual/runtime/primitives.md

```diff
@@ -62,7 +61,7 @@ Runtime Module Library (SRML):
 
 ### Learn More
 
-- Learn about the [Substrate Runtime Module Library](conceptual/runtime/srml.md).
+- Learn about the [Substrate FRAME](conceptual/runtime/frame.md).
```

Comment:
This requires to change the name of the file to `conceptual/runtime/frame.md`
It may be worth to track if there are links pointing to it.

Reply:
Yeah, checked it already in the devhub repo.

Reply:
Does the `frame.md` file exists?

Reply:
`srml.md` is renamed to `frame.md` in this PR.

Reply:
Nice! I didn't noticed the change. Now I saw it at the top :+1: